CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### 5.1.0 (Sep 2023)
+
+* [UPDATE] Update `System.Threading.Tasks.Extensions` to latest (`4.5.4-*`) as per @Romfos' suggestion. (#731)
+* [UPDATE] Update `Castle.Core` to `5.1.1-*` to support C# 9 covariants. Thanks @siblount to tracking this down. (#730)
+* [UPDATE] Improved support for testing `ILogger`. Thanks to @zlanger for this contribution, and also thanks to @Saibamen for reviewing this PR. (#732)
```

Comment:
```suggestion
* [UPDATE] Improved support for testing `ILogger`. Thanks to @zlangner for this contribution, and also thanks to @Saibamen for reviewing this PR. (#732)
```

Reply:
So sorry! I thought I copy pasted the names to avoid that mistake, no idea how I managed to stuff that up. 😓🤦

Thanks for picking this up!

Reply:
don't sweat it! it's a very common mistake